test/WebJobs.Script.Tests.Integration/ApplicationInsights/ApplicationInsightsEndToEndTestsBase.cs

```diff
@@ -0,0 +1,144 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Logging;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.ApplicationInsights
+{
+    public abstract class ApplicationInsightsEndToEndTestsBase<TTestFixture> :
+        EndToEndTestsBase<TTestFixture> where TTestFixture : ApplicationInsightsTestFixture, new()
+    {
+        private ApplicationInsightsTestFixture _fixture;
+
+        public ApplicationInsightsEndToEndTestsBase(TTestFixture fixture) : base(fixture)
+        {
+            _fixture = fixture;
+        }
+
+        protected async Task ApplicationInsights_SucceedsTest()
+        {
+            string functionName = "Scenarios";
+            TestHelpers.ClearFunctionLogs(functionName);
+
+            string guid = Guid.NewGuid().ToString();
+
+            ScenarioInput input = new ScenarioInput
+            {
+                Scenario = "appInsights",
+                Container = "scenarios-output",
+                Value = guid
+            };
+
+            Dictionary<string, object> arguments = new Dictionary<string, object>
+            {
+                { "input", JsonConvert.SerializeObject(input) }
+            };
+
+            await Fixture.Host.CallAsync(functionName, arguments);
+
+            // make sure file logs have the info
+            IList<string> logs = null;
+            await TestHelpers.Await(() =>
+            {
+                logs = TestHelpers.GetFunctionLogsAsync(functionName, throwOnNoLogs: false).Result;
+                return logs.Count > 0;
+            });
+
+            // No need for assert; this will throw if there's not one and only one
+            logs.Single(p => p.EndsWith(guid));
+
+            Assert.Equal(10, _fixture.TelemetryItems.Count);
```

Comment:
Right now this test fails but I have a fix coming for the SDK that fixes it -- will send first thing Monday.